libraries/cmake/facebook/modules/api.cmake

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) 2014-present, Facebook, Inc.
```

Comment:
The path `libraries/cmake/facebook/modules/api.cmake` doesn't make a lot of sense.

Alternatively, move this logic into `/cmake/`

Reply:
I don't feel strongly about this. Your call on where `api` and `utils` are located.

Reply:
I have stored the APIs there since they are layer-specific. Since the libraries folder (at least on the CMake side) is self-contained and not used directly by the main project, I ended up saving the file near the entry point (the find_package handlers).

We can move it elsewhere, keeping in mind however that the FindPackage script should keep using relative paths